dspy/predict/avatar/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .avatar import *
```

Comment:
not sure about the repo structure here so tagging @okhat - would it be better to move all avatar related files to [dspy/experimental](https://github.com/stanfordnlp/dspy/tree/main/dspy/experimental)?

Reply:
would be great to eventually have an abstract `agent.py` class in predict then